compiler/src/iree/compiler/Dialect/LinalgExt/Transforms/Passes.td

```diff
@@ -50,6 +50,17 @@ def TopkSplitReductionPass:
   ];
 }
 
+def DecomposeAggregatedOpPass :
+    InterfacePass<"iree-linalg-ext-decompose-aggregated-ops", "mlir::FunctionOpInterface"> {
+  let summary = "Decomposes ops that implement the AggregatedOpImpl interface";
+  let options = [
+    Option<"filterOps", "filter-ops", "std::string", /*default=*/"",
+           "If specified, filters such that only ops in the comma-delimited string are decomposed.
+                          Op names must be fully-qualified, including the dialect name."
+           "If unspecified, filters all ops by default.">
```

Comment:
Could you change the default to not decompose anything if nothing is specified. Its easier to control the pass that way.